app/Actions/Service/DeleteService.php

```
@@ -55,6 +57,23 @@ public function handle(Service $service, bool $deleteVolumes, bool $deleteConnec
         } catch (\Exception $e) {
             throw new \RuntimeException($e->getMessage());
         } finally {
+            try {
+                app(EdgeProxyRemoteRouteService::class)->deleteService($service);
+            } catch (\Throwable $exception) {
+                Log::warning('Failed to delete edge proxy route file for service.', [
+                    'service_uuid' => $service->uuid,
+                    'error' => $exception->getMessage(),
+                ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'handle'. (undefined)

(StaticAccess)